cmd/tckconvert.cpp

```diff
@@ -102,41 +109,76 @@ void usage ()
 
 class VTKWriter: public WriterInterface<float> { MEMALIGN(VTKWriter)
```

Comment:
Having the boolean switch between ASCII and binary as a template parameter rather than a constructor input would be theoretically more efficient. Speculative execution will make up some of the ground, but has famously led to its own issues... nevertheless, performance is maybe not a concern here, but for #411 would want to template it out.

Reply:
It'll be mostly inheritance, I suppose in retrospect it would depend on the amount of shared code between VTK ASCII vs. binary as to whether to have them as two templated versions of the same derived class or two different classes.